doc/releases/changelog-dev.md

```diff
@@ -31,6 +31,8 @@
 * Circuit cutting now performs expansion to search for wire cuts in contained operations or tapes.
   [(#2340)](https://github.com/PennyLaneAI/pennylane/pull/2340)
 
+* The `QuantumTape` now processes all `Operator` objects into `_ops` list, instead of just `Operation` objects.
```

Comment:
```suggestion
* The `QuantumTape` now processes all `Operator` objects into `_ops` list, instead of just `Operation` objects.
  [(#2353)](https://github.com/PennyLaneAI/pennylane/pull/2353)
```